src/app/src/test/kotlin/org/vaccineimpact/api/tests/controllers/ModellingGroupControllersTests.kt

```diff
@@ -364,6 +364,43 @@ class ModellingGroupControllersTests : ControllerTests<ModellingGroupController>
         controller.modifyMembership(context, mock<UserRepository>())
     }
 
+    @Test
+    fun `can return touchstones list if user has permission to read prepared touchstones`()
+    {
```

Comment:
The name of these tests could be a little more descriptive - e.g. this one could be "returns in preparation touchstones if user has..." and the one below "only returns open touchstones if user has no permission...."